codecov_cli/helpers/ci_adapters/cloudbuild.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from codecov_cli.helpers.ci_adapters.base import CIAdapterBase
+
+
+class GoogleCloudBuildCIAdapter(CIAdapterBase):
+    # https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values
+
+    def detect(self) -> bool:
+        return (
+            bool(os.getenv("LOCATION"))
+            and bool(os.getenv("PROJECT_NUMBER"))
+            and bool(os.getenv("PROJECT_ID"))
+            and bool(os.getenv("BUILD_ID"))
+        )
+
+    def _get_branch(self):
+        return os.getenv("BRANCH_NAME")
+
+    def _get_build_code(self):
+        return os.getenv("BUILD_ID")
+
+    def _get_commit_sha(self):
+        return os.getenv("COMMIT_SHA")
+
+    def _get_slug(self):
+        return os.getenv("REPO_FULL_NAME")
+
+    def _get_build_url(self):
+        return None
```

Comment:
It does not exist as a variable, but it can be built from 3 of the required variables, like this:
https://console.cloud.google.com/cloud-build/builds;region=$LOCATION/$BUILD_ID?project=$PROJECT_ID
And this is how I plan to implement it.

Reply:
implemented